velodyne_driver/src/driver/driver.cc

```diff
@@ -92,7 +95,8 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
     }
   else
     {
-      input_.reset(new velodyne_driver::InputSocket(private_nh));
+      input_.reset(new velodyne_driver::InputSocket(private_nh,
+                                                    config_.udp_port));
```

Comment:
Why pass ``config_.udp_port`` explicitly? Isn't the whole ``config_`` structure already available?